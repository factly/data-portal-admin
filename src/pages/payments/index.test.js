import React from 'react';
import { shallow } from 'enzyme';

import PaymentList from './index';

describe('Payment List component', () => {
  it('should render the component', () => {
    const component = shallow(<PaymentList />);
    expect(component).toMatchSnapshot();
  });
});
