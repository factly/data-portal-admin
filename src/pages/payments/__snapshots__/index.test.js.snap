// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment List component should render the component 1`] = `<PaymentsList />`;
